mlflow/tracking/__init__.py

```diff
@@ -411,3 +411,39 @@ def _get_git_commit(path):
         return commit
     except (InvalidGitRepositoryError, GitCommandNotFound, ValueError, NoSuchPathError):
         return None
+
+
+def _get_git_url(uri):
+    """
+    Return the git repo path github_url#sub_directory from the uri input
+    :param uri: the expanded uri
+    :return: the git repo path included sub directory if the uri is part of git repo,
+             otherwise return the input as output
+    """
+    if '#' in uri:
+        # already a URI in git repo format
```

Comment:
Capitalize comments to follow the existing code style

Reply:
Done